Formula/freshenv.rb

```diff
@@ -0,0 +1,89 @@
+class Freshenv < Formula
+  include Language::Python::Virtualenv
+
+  desc "Provision and manage local developer environments"
+  homepage "https://github.com/raiyanyahya/freshenv"
+  url "https://files.pythonhosted.org/packages/88/91/75539ca1a7b0aec86dc5759be1a6167edc017e950beecd1d67f3018587d6/freshenv-0.1.2.tar.gz"
+  sha256 "9cbed9143cb904b5fa5c2f226ecd95a9f9b5692cee773b85cfb402020176d273"
+  license "MIT"
+  depends_on "python@3.8"
```

Comment:
```suggestion
  license "MIT"

  depends_on "python@3.8"
```
Does this not work with a newer Python?